### DISCUSSION TITLE
建议添加RIFE性能需求文档和新的多档RIFE性能档位

### DISCUSSION THREAD

**Discussion by o5-null** [hooke007/mpv_PlayKit#490] [mpv 通用]
如题
测试了ncnn版本和cuda版本
性能差距巨大
ncnn在2080s下最高仅能做到1080p48hz
而cuda版本可以做到2160p72hz
建议添加多个性能档位
默认的ncnn过于激进
而cuda过于保守

**Comment by hooke007** [comment]
你看说明了吗。。。
ncnn默认都跑不了还要加啥档位，已经性能最低了

**Reply by o5-null** [reply]
std默认是1440啊
太高了
设到720就行

**Reply by hooke007** [reply]
H_pre 是压限用的
不是所有人都需要限制

**Reply by hooke007** [reply]
ncnn是给用不了trt的显卡做的，能跑trt的没有跑ncnn的必要
